custom_components/kumo/__init__.py

```diff
@@ -1,121 +1,46 @@
-"""Support for Mitsubishi KumoCloud devices."""
+"""
```

Comment:
Please explain your changes to this file. It's completely unrecognizable and no longer mentions Kumo or pykumo at all. It looks like generic example code? Did you forget to 'git add' some new files, perhaps?

Reply:
see new init file below https://github.com/dlarrick/hass-kumo/pull/19/commits/8ff475a8fa90c6599f8c330526443e3271428af6